RecoTracker/DeDx/plugins/DeDxHitCalibrator.cc

```diff
@@ -0,0 +1,426 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackDeDxHits.h"
+#include "DataFormats/TrackReco/interface/DeDxHit.h"
+#include "DataFormats/TrackReco/interface/DeDxHitInfo.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/SiStripCommon/interface/ConstantsForHardwareSystems.h"
+#include "CondFormats/PhysicsToolsObjects/interface/DeDxCalibration.h"
+#include "CondFormats/DataRecord/interface/DeDxCalibrationRcd.h"
+#include "CondFormats/SiPixelObjects/interface/PixelIndices.h"
+
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
+#include "Geometry/CommonDetUnit/interface/PixelGeomDetUnit.h"
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+#include "Geometry/TrackerGeometryBuilder/interface/StripGeomDetUnit.h"
+#include "CalibTracker/SiPixelESProducers/interface/SiPixelGainCalibrationOfflineService.h"
+
+#include <fstream>
+
+class DeDxHitCalibrator : public edm::stream::EDProducer<> {
+public:
+  enum { kIsNormal, kIsBelow, kIsOver };
```

Comment:
```suggestion
  static constexpr int kIsNormal = 0, kIsBelow = 1, kIsOver = 2;
```
(see e.g. https://github.com/cms-sw/cmssw/pull/45133)

Reply:
Addressed in 05c2dc85edef1e8e3a387cd8694941aa4c4e24bd